Gems/PhysX/Common/3rdParty/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+include(FetchContent)
+
```

Comment:
I'd recommend adding
```cmake
if (TARGET 3rdParty::v-hacd)
  return()
endif()
```

Just to allow people who want to substitute their own v-hacd for some reason to be able to do so before this executes.